oauthenticator/__init__.py

```diff
@@ -1,3 +1,4 @@
 from .oauth2 import *
 from .github import *
 from .bitbucket import *
+from .mediawiki import *
```

Comment:
My underlying concern is that there'll be inconsistency between oauthenticator.GitHubOAuthenticator vs oauthenticator.mediawiki.MWOauthenticator.